nano/node/network.cpp

```diff
@@ -215,15 +215,21 @@ bool nano::network::send_votes_cache (std::shared_ptr<nano::transport::channel>
 	return result;
 }
 
-void nano::network::flood_message (nano::message const & message_a)
+void nano::network::flood_message (nano::message const & message_a, bool const & is_dropable)
```

Comment:
Minor typo `is_droppable`. Passing a bool by ref won't be faster, primitives are generally passed by value. Also it should probably have the `_a` suffix to be consistent with the existing parameter. Same in `flood_block`

Reply:
thanks :)